app/src/test/java/org/commcarehq/aadharuid/ScanResultTest.java

```diff
@@ -169,4 +170,29 @@ public void testMisplacedQuestionMark() {
+        ScanResult scanResult = new ScanResult(rawString);
+        assertEquals(scanResult.statusCode, 0);
+        assertEquals(scanResult.statusText, "✓");
+        assertEquals(scanResult.rawString, rawString);
+        assertEquals(scanResult.uid, "xxxxxxxx8908");
+        assertEquals(scanResult.name, "Penumarthi Venkat");
```

Comment:
Can you confirm this is the test ID distributed by Aadhaar and not a real person's ID?

Reply:
That's correct!

Reply:
Thanks for confirming (It might be worth a note to mention it in code given the sensitivity of the data)